public/css/freelancer.css

```diff
@@ -208,7 +215,12 @@ ul.tag-list li {
 #freelancer-claim-form {
   background-color: ghostwhite;
   display: flex;
-  flex-flow: column;
+  flex-wrap: wrap;
+  justify-content: flex-start
+  flex-flow: row;
```

Comment:
row is the default value of flex-flow, so it can be removed.
your `background-color: ghostwhite` gets overridden (you can remove it), justify-content breaks your code, because you forgot a semicolon.